pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -321,15 +321,18 @@ public CompletableFuture<Void> initialize() {
                         isEncryptionRequired = false;
                         updatePublishDispatcher();
                         updateResourceGroupLimiter(optPolicies);
-                        initializeRateLimiterIfNeeded();
+                        initializeDispatchRateLimiterIfNeeded();
+                        updateSubscribeRateLimiter();
```

Comment:
Before this, we forget to update `updateSubscribeRateLimiter` ?

Reply:
No, before this, the `updateSubscribeRateLimiter ` is inside the  `initializeDispatchRateLimiterIfNeeded `

Reply:
ah. ok.